mne_bids/mne_bids.py

```diff
@@ -685,7 +685,8 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     channels_fname = make_bids_basename(
         subject=subject_id, session=session_id, task=task, run=run,
         acquisition=acquisition, suffix='channels.tsv', prefix=data_path)
-    if ext not in ['.fif', '.ds', '.vhdr', '.edf', '.bdf', '.set']:
+    if ext not in ['.fif', '.ds', '.vhdr', '.edf', '.bdf', '.set', '.con',
+                   '.sqd']:
```

Comment:
we should eventually just change it to `ext in blah`. This is so confusing ...

Reply:
the only ext now that should be in that list is `.pdf` right? Just wanting to finish off this PR...

Reply:
I think so ...